src/ui/projects/CreateProjectPage.js

```diff
@@ -0,0 +1,164 @@
+import React, { useCallback, useState, useContext } from "react";
+import PropTypes from "prop-types";
+import ScrollToTop from "../router/ScrollToTop";
+import NavBar from "../navigation/NavBar";
+import {
+  ProjectGrid,
+  ProjectGridContainer,
+  ProjectGridHeader,
+  ProjectGridHeaderRow,
+  Filter,
+  Separator,
+  SearchInput,
+  ProjectGridContent,
+  ErrorMessage
+} from "./ProjectGrid";
+import Footer from "../navigation/Footer";
+import PrimaryLink from "../inputs/PrimaryLink";
+import { Button } from "../inputs/Button";
+import { ProjectsSection, ProjectsContainer, ProjectsHeader } from "./ProjectsPage";
+import { ApiContext } from "../contexts/ApiContext";
+import { Link } from "react-router-dom";
+import InfiniteScroll from "react-infinite-scroller";
+import usePaginatedSearch from "./usePaginatedSearch";
+
+export default function CreateProjectPage({ history, location }) {
+  const api = useContext(ApiContext);
+
+  const queryParams = new URLSearchParams(location.search);
+
+  const [params, setParams] = useState({
+    source: "scene_listings",
+    filter: queryParams.get("filter") || "featured-remixable",
+    q: queryParams.get("q") || ""
+  });
+
+  const updateParams = useCallback(
+    nextParams => {
+      const search = new URLSearchParams();
+
+      for (const name in nextParams) {
+        if (name === "source" || !nextParams[name]) {
+          continue;
+        }
+
+        search.set(name, nextParams[name]);
+      }
+
+      history.push(`/projects/create?${search}`);
+
+      setParams(nextParams);
+    },
+    [history]
+  );
+
+  const onChangeQuery = useCallback(
```

Comment:
Seems weird you can only text search "all", but in checking it seems this is in line with how other media searches work...

Reply:
Yeah, I figured that was the UX we wanted since we do it elsewhere. I'm not sure you usually want to search featured remixable scenes. Most of the time you want to search all remixable scenes.